index.js

```diff
@@ -126,26 +127,59 @@ function validate(word) {
   return true;
 }
 
-function stonksify(word) {
+function stonksify(word, showAllWords = true) {
   word = word.toLowerCase();
+  let possibleWords = [];
   console.groupCollapsed("Regex Details: " + word);
   for (let i = 0; i < stonksifyRules.length; i++) {
     let toReplace = new RegExp(stonksifyRules[i][0]);
     let replaceWith = stonksifyRules[i][1];
     let newWord = word.replace(toReplace, replaceWith);
+    if (possibleWords.indexOf(newWord) === -1) {
+      possibleWords.push(newWord);
+    }
     console.log(
-      `Stonksify rule ${i}: replace regexp ${toReplace} + with '${replaceWith}.`
+      `Stonksify rule ${i} (${newWord}): replace regexp ${toReplace} + with '${replaceWith}.`
     );
     word = newWord;
   }
   console.groupEnd();
+  if (showAllWords) {
+    displayPossibleWords(possibleWords);
```

Comment:
this should be moved outside the function. I recommend having stonksfiy return an array of stonked words and then adding to the possible words list after this function returns. We don't want this one function performing two things (stonking and updating possible words on screen)